src/encoded/genomic_data_service.py

```diff
@@ -111,6 +113,28 @@ def rna_get_request(self, format_='json'):
                 facet['total'] += 1
 
 
+    def rna_get_autocomplete(self):
+        suggestions = []
+
+        if self.autocomplete_query:
+            try:
+                url = f'{self.path}{RNA_GET_AUTOCOMPLETE}?gene={self.autocomplete_query}'
+
+                results = requests.get(url, timeout=3).json()
+
+                if 'genes' in results:
+                    suggestions = [{'text': result} for result in results['genes']]
+            except:
```

Comment:
Code-smell. Why catch an exception but not use it? It makes debugging harder.

Reply:
for this scenario, in case of any of the multiple possible exceptions from the `requests` call, we should just ignore and return an empty result back to the client. We won't just show any autocomplete results in case of failure

Reply:
Fair enough.